holidays/locale/en_US/LC_MESSAGES/AF.po

```diff
@@ -0,0 +1,98 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# Afghanistan holidays en_US localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.64\n"
+"POT-Creation-Date: 2024-12-24 13:40+0530\n"
+"PO-Revision-Date: 2024-12-31 15:52+0700\n"
+"Last-Translator: Prateekshit Jaiswal <prateekshitjaiswal73@gmail.com>\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.4\n"
+"X-Generator: Poedit 3.5\n"
+
+#. Liberation Day.
+msgid "روز آزادی"
+msgstr "Liberation Day"
+
+#. Nowruz.
+msgid "نوروز"
+msgstr "Nowruz"
+
+#. International Workers' Day.
+msgid "روز جهانی کارگر"
+msgstr "International Workers' Day"
+
+#. Martyrs' Day.
+msgid "روز شهیدان"
+msgstr "Martyrs' Day"
+
+#. Day of Arafah.
+msgid "روز عرفه"
+msgstr "Day of Arafah"
+
+#. Ashura.
+msgid "عاشورا"
+msgstr "Ashura"
+
+#. Prophet's Birthday.
+msgid "میلاد پیامبر"
+msgstr "Prophet's Birthday"
+
+#. First Day of Ramadan.
+msgid "اول رمضان"
+msgstr "First Day of Ramadan"
+
+#. %s (estimated).
+#, c-format
+msgid "%s (برآورد شده)"
+msgstr "%s (estimated)"
+
+#. %s (observed).
+#, c-format
+msgid "%s (مشاهده شده)"
+msgstr "%s (observed)"
+
+#. %s (observed, estimated).
+#, c-format
+msgid "%s (مشاهده شده، برآورد شده)"
+msgstr "%s (observed, estimated)"
+
+#. Soviet Victory Day.
+msgid "روز پیروزی شوروی"
+msgstr "Soviet Victory Day"
+
+#. Mujahideen Victory Day.
```

Comment:
Unless someone changed around the codes again, my only recommendation left is to match the comments in Afghanistan main file with `en_US` loc for both "Mojahedin's Victory Day" and  "The Day of America's Withdrawal" day.

This is otherwise good to go :pray: